Packs/DelineaDSV/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DelineaDSV",
+    "description": "This integration pack helps you retrieve the data stored in the Delinea DevOps Storage Vault and use it in other integrations.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Delinea Inc",
+    "url": "https://delinea.com/support",
+    "email": "support@delinea.com",
+	"created": "2022-08-10T08:04:48Z",
+    "categories": [
+        "Authentication"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+	"githubUser": [
+        "RoshaniTambadkar"

```

Comment:
is this (and [here](https://github.com/demisto/content/pull/20852/files#diff-257c3c76800528cc0258d607ef3142b0bfb602de7f69419b39fcd692bb704602R17), too) the github user mentioned here (and not @tylerezimmerman) on purpose?

Reply:
@dorschw that is the member of my engineering team that did the work on this, I was just the PR facilitator. Should my name be added as well?

Reply:
It doesn't have to be yours, just of someone from the organization.